scrnaseq.Rmd

```diff
@@ -16,6 +16,87 @@ output:
 bibliography: "references.bib"
```

Comment:
General: Shall we move `fig.asp` to some issue for after our release? It doesn't seem that important.

Reply:
Done